state/account.go

```diff
@@ -29,10 +29,10 @@ var (
 	ErrUnknownAccountType = errors.New("unknown account type")
 )
 
-// ZeroNonceAccountTypeOption is an option to create account with new account type
-func ZeroNonceAccountTypeOption() AccountCreationOption {
+// LegacyNonceAccountTypeOption is an option to create account with new account type
+func LegacyNonceAccountTypeOption() AccountCreationOption {
 	return func(account *Account) error {
-		account.accountType = 1
+		account.accountType = 0
```

Comment:
as commented above, can we still use `ZeroNonceAccountTypeOption`? it is more natural, no option corresponds to default case (legacy account)